src/rust/graph-descriptions/proto/graph_description.proto

```diff
@@ -9,9 +9,9 @@ message Session {
     // Such as the pid + assetid
     repeated string primary_key_properties = 1;
     bool primary_key_requires_asset_id = 2;
-    uint64 created_time = 3;
-    uint64 last_seen_time = 4;
-    uint64 terminated_time = 5;
+    string created_time = 3;
```

Comment:
These are no longer representing the timestamp, but instead which field might hold the timestamp. It's primarily helpful from a code generation perspective. I could take some time to try to get it working with integers, so that we don't carry around strings just to index into data we already understand, but I'm not particularly skilled at writing proc macros.